github-search/sample_github_search.py

```diff
@@ -3,18 +3,46 @@
 """
 from github import Github
 import github_token as access
+import csv
+import json
 
 def get_repo_stars(repository):
     "Return the number of stars a repo contains"
     github_obj = Github(access.TOKEN)
     repo = github_obj.get_repo(repository)
     stars = repo.stargazers_count
 
-    return stars 
+    return stars
+
+def get_repo_forks(repository):
+    "Return the number of forks a repo contains"
+    github_obj = Github(access.TOKEN)
+    repo = github_obj.get_repo(repository)
+    forks = repo.forks_count
+
+    return forks 
+
+def get_repo_open_issues(repository):
+    "Return the number of watchers a repo contains"
+    github_obj = Github(access.TOKEN)
+    repo = github_obj.get_repo(repository)
+    open_issues = repo.open_issues_count
+
+    return open_issues 
 
 #----START OF SCRIPT
 if __name__ == '__main__':
-    repositories = ['techgaun/github-dorks','UnkL4b/GitMiner','michenriksen/gitrob','PyGithub/PyGithub']
+    repositories = ['qxf2/qxf2-page-object-model']
     for repository in repositories:
         stars = get_repo_stars(repository)
+        forks = get_repo_forks(repository)
+        open_issues = get_repo_open_issues(repository)
         print('{} has {} stars'.format(repository,stars))
+        print('{} has {} forks'.format(repository,forks))
+        print('{} has {} open_issues'.format(repository,open_issues))
+        ## Write API Results to CSV
+        with open('repo_count.csv', 'wb') as csvFile:
```

Comment:
If you open in 'write' mode, wouldn't you end up over-writing the existing data every time? You should be opening in 'append' mode and write only the data.